### ISSUE TITLE
blimpy.waterfall WARNING  Could not import bitshuffle, using hdf5plugin failover. Write access disabled.

### ISSUE THREAD

**Issue by cmorsoc** [UCBerkeleySETI/blimpy#113]
Hi, I'm running the blimpy tutorial and we get this warning:

> blimpy.waterfall WARNING  Could not import bitshuffle, using hdf5plugin failover. Write access disabled.

When running these lines:

> import pylab as plt
> from blimpy import Waterfall 

And:

> /home/cmorsoc/.local/lib/python3.7/site-packages/blimpy/io/file_wrapper.py:323: H5pyDeprecationWarning: The default file mode will change to 'r' (read-only) in h5py 3.0. To suppress this warning, pass the mode you need to h5py.File(), or set the global default h5.get_config().default_file_mode, or set the environment variable H5PY_DEFAULT_READONLY=1. Available modes are: 'r', 'r+', 'w', 'w-'/'x', 'a'. See the docs for details.

  self.h5 = h5py.File(self.filename)
When running:

> file_path = 'Voyager1.single_coarse.fine_res.h5'
> obs = Waterfall(file_path)

**Setup**
 - Python version: 3.X
 - Blimpy version? Latest (pip3 install blimpy)

**Comment by telegraphic** [comment]
Thanks @cmorsoc -- in the short term, these are safe to ignore in terms of functionality, but longer term we should change the code so the warnings don't get raised.

The first is a warning that the `bitshuffle` package isn't installed. A lot of people have had issues installing it, so we started using a package `hdf5plugin`. Early versions of `hdf5plugin` couldn't write out bitshuffle-compressed datasets, but this might have been added, so time to revisit.

The second is a warning from `h5py`, that in future versions they will open files as read-only by default. This might break stuff for other people who rely on its old behaviour, hence the message. This is also easy enough to fix, just adding `mode='r'` to a few calls.
